test/integration/overload_integration_test.cc

```diff
@@ -456,6 +512,24 @@ class LoadShedPointIntegrationTest : public BaseOverloadIntegrationTest,
     initialize();
     updateResource(0);
   }
+  void
+  initializeWithBypassOverloadManager(const envoy::config::overload::v3::LoadShedPoint& config) {
+    setupOverloadManagerConfig(config);
+    config_helper_.addConfigModifier([this](envoy::config::bootstrap::v3::Bootstrap& bootstrap) {
+      *bootstrap.mutable_overload_manager() = this->overload_manager_config_;
+    });
+    config_helper_.addConfigModifier([](envoy::config::bootstrap::v3::Bootstrap& bootstrap) {
+      auto* listener = bootstrap.mutable_static_resources()->mutable_listeners(0);
+      listener->set_bypass_overload_manager(true);
+      // add a listener that doesn't bypass overload manager.
+      auto* new_listener = bootstrap.mutable_static_resources()->add_listeners();
+      new_listener->CopyFrom(*listener);
+      new_listener->set_name("http_2");
+      new_listener->set_bypass_overload_manager(false);
+    });
+    initialize();
+    updateResource(0);
+  }
```

Comment:
may have forgotten to port `TEST_P(LoadShedPointIntegrationTest, HttpConnManagerDoesNotShedLoadWhenBypassed)`?